### ISSUE TITLE
Reflection warnings

### ISSUE THREAD

**Issue by seancorfield** [RutledgePaulV/newrelic-clj#1]
Not sure how easy these are to fix but wanted to let you know about them:

```
Reflection warning, io/github/rutledgepaulv/newrelic_clj/api.clj:130:15 - call to method putAll can't be resolved (target class is unknown).
Reflection warning, io/github/rutledgepaulv/newrelic_clj/api.clj:227:8 - reference to field convertToWebTransaction can't be resolved.
Reflection warning, io/github/rutledgepaulv/newrelic_clj/api.clj:233:8 - reference to field convertToWebTransaction can't be resolved.
Reflection warning, io/github/rutledgepaulv/newrelic_clj/inject.clj:68:28 - reference to field close can't be resolved.
Reflection warning, io/github/rutledgepaulv/newrelic_clj/inject.clj:80:13 - reference to field close can't be resolved.
Reflection warning, io/github/rutledgepaulv/newrelic_clj/inject.clj:94:13 - reference to field close can't be resolved.
Reflection warning, io/github/rutledgepaulv/newrelic_clj/internals.clj:11:20 - call to static method alength on clojure.lang.RT can't be resolved (argument types: unknown).
```

**Comment by RutledgePaulV** [comment]
Fixed in `1.0.2`

**Comment by seancorfield** [comment]
Thanks!
